src/elements/tag/tag-group/tag-group.ts

```diff
@@ -91,7 +91,7 @@ class SbbTagGroupElement extends SbbNamedSlotListMixin<SbbTagElement, typeof Lit
         ? this.tags.filter((t) => t.checked).map((t) => t.value)
         : (this.tags.find((t) => t.checked)?.value ?? null);
   }
-  private _value: string | string[] | null = null;
+  private _value: string | (string | null)[] | null = null;
```

Comment:
Can that be happen, an array consisting of null?

Reply:
`value` comes from the form-associated mixin, and it's nullable, so technically yes if all the tags have no value